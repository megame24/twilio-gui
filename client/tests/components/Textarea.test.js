import React from 'react';
import { shallow } from "enzyme";
import Textarea from '../../src/components/Textarea';

describe('Testing Textarea component', () => {
  it('Should render as expected', () => {
    const tree = shallow(<Textarea />);
    expect(tree).toMatchSnapshot();
  });
});
