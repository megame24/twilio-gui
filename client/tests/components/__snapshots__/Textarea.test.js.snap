// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Textarea component Should render as expected 1`] = `<textarea />`;
